caddy.go

```diff
@@ -836,7 +839,7 @@ var (
 	instancesMu sync.Mutex
 )
 
-const (
+var (
```

Comment:
> Ok, but this is a regression for me. How can we/I make it so that if -type=dns
is will look for a Corefile instead of a Caddyfile?

Set the DefaultInput field of your server type to a function that loads a file called Corefile, perhaps? But if there is a Caddyfile in the directory already, then it will use that anyway... :-/